fe-vue/ocp.env

```diff
@@ -0,0 +1,2 @@
+MEMORY_LIMIT=256Mi
```

Comment:
I think all the `fe-*` ones can actually have a `MEMORY_LIMIT=128Mi` as they use Nginx and will likely not need that much memory ...

Reply:
You are suggesting limit and request being 128 for fe-* right?
@akhilsoman can you take a look?

Reply:
I added the parameter to the other Jenkinsfile(s) now as well.
Regarding changes for the memory limit I wanted to wait for @akhilsoman's input.
On the other hand if the fe quickstarters never use that much memory, there should be no difference to the cluster as they presumably would only use the requested value.

Reply:
As @michaelsauter said, all the fe-* quickstarters uses Nginx to serve static content and  it's unlikely that they will need more than 128. But in my opinion, we could keep the limit at 256Mi as the additional memory will not be used from the cluster, unless needed by the container.

Reply:
@henrjk Your call.

Let me know when this PR is final for review/merge.